releases/v3.0.0.toml

```diff
@@ -0,0 +1,2930 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "registry"
+github_repo = "distribution/distribution"
+
+# previous release
+previous = "v2.8.3"
+
+pre_release = false
+
+preface = """\
+Welcome to the 3.0.0 release of registry!
```

Comment:
This should mention that it's a pre-release instead? (similar to https://github.com/containerd/containerd/releases/tag/v2.0.0-beta.0).